### PR TITLE
Aws

### PR DESCRIPTION
**Pull request by ibilalkayy**
## Changes proposed

Added the AWS content and did some modifications to the file.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5090"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SurajPratap10 — APPROVED** [review]
good

File: data/ibilalkayy.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "Bilal Khan",
   "type": "personal",
-  "bio": "College Dropout ➞ Self-Taught DevOps Guy | Open Source Contributor to Kubernetes, Content Creator on YouTube & a Technical Writer at Hashnode.",
+  "bio": "College Dropout ➞ Self-Taught DevOps Guy | AWS Community Builder, Open Source Contributor, Content Creator on YouTube & Hashnode.",
 
-  "tags": ["Open Source", "DevOps", "Golang", "Kubernetes", "Docker", "DevRel"],
+  "tags": ["Open Source", "DevOps", "Golang", "DevRel"],
```
Comment (WillBeebe) [inline data/ibilalkayy.json:6]:
```suggestion
  "tags": ["OpenSource", "DevOps", "Golang", "DevRel"],
```
📋 tags usually have to _not_ contain spaces. Not sure if EddieHub ™️ supports tags with spaces, I'd have to paw around the code. However, it's a good practice in general when you see "tag", to just skip the spaces. Consistency in that way makes it a lizard brain task. 🤙 

Also, if I said "containerize this npm module / Rust thing" would you know what that means / how to do it? If so, the 2023 version of Docker is `OCIContainers`. Toss that in there and you'll have yacht-money in no-time.

File: data/ibilalkayy.json [hunk 2]
```diff
@@ -78,6 +78,22 @@
       "description": "Started creating content",
       "url": "https://www.youtube.com/@ibilalkayy/"
     },
+    {
+      "title": "Started Hashnode",
+      "date": "Aug, 2022",
+      "icon": "SiHashnode",
+      "color": "blue",
+      "description": "Started writing blogs",
+      "url": "https://hashnode.com/@ibilalkayy"
```
Comment (WillBeebe) [inline data/ibilalkayy.json:87]: Plus one follower 🙋

**Review by WillBeebe — COMMENTED** [review]
Coupla notes, otherwise LGTM.

File: data/ibilalkayy.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "Bilal Khan",
   "type": "personal",
-  "bio": "College Dropout ➞ Self-Taught DevOps Guy | Open Source Contributor to Kubernetes, Content Creator on YouTube & a Technical Writer at Hashnode.",
+  "bio": "College Dropout ➞ Self-Taught DevOps Guy | AWS Community Builder, Open Source Contributor, Content Creator on YouTube & Hashnode.",
```
[the remark this answers is not in the document]
Comment (ibilalkayy) [reply data/ibilalkayy.json:4]: I don't think that a big bio is required.

File: data/ibilalkayy.json (hunk 1 above)
Comment (priyankarpal) [reply data/ibilalkayy.json:6]: I think it supports :)

image

**Review by priyankarpal — APPROVED** [review]

**Review by amandamartin-dev — APPROVED** [review]
LGTM!

**Pull request merged**